src/lib/Scenes/Home/Components/Sales/Components/SaleListItem.tsx

```diff
@@ -69,13 +69,17 @@ const Metadata = styled.Text`
   font-size: 10px;
 `
 
-export class SaleListItem extends React.Component<RelayProps, null> {
+interface Props extends RelayProps {
+  screenWidth: number
+}
+
+export class SaleListItem extends React.Component<Props, null> {
   get containerWidth(): number {
-    const screenSize = Dimensions.get("window")
-    const isIPad = screenSize.width > 700
+    const screenWidth = this.props.screenWidth
+    const isIPad = screenWidth > 700
     const numColumns = isIPad ? 4 : 2
     const gutterSize = isIPad ? 100 : 60
-    return (screenSize.width - gutterSize) / numColumns
+    return (screenWidth - gutterSize) / numColumns
   }
```

Comment:
I think all of this should move out of this component up to the parent (`SaleList`), there’s no need for this component to know anything about number of columns, also:

* moving it up means we only need to calculate the container width _once_
* the parent can just use standard `style` API to set the width, rather than something related to ‘screen’